### PR TITLE
Updated Version EllipticFEM v0.0.2.

### PR DESCRIPTION
**Pull request by gerhardtulzer**

### CODE REVIEW

**Comment by tkelman** [comment]
what's the minimum supported julia version?

**Comment by ararslan** [comment]
I tried to provide some help in this issue I opened on your project: https://github.com/gerhardtulzer/EllipticFEM.jl/issues/5

**Comment by gerhardtulzer** [comment]
Hi there!
Is the update now OK, or do I need to change anything else?

**Comment by tkelman** [comment]
It's missing the requires file in this pull request

**Comment by kmsquire** [comment]
Bump

**Comment by gerhardtulzer** [comment]
OK, I added the requires file.

File: EllipticFEM/versions/0.0.2/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+f5284cfea605d036c8c161b675af77a14480da2f
```
Comment (tkelman) [inline EllipticFEM/versions/0.0.2/sha1]: this sha doesn't match the git tag - which version do you want the release to use?

**Comment by gerhardtulzer** [comment]
OK, I'm now entirely confused. I want to release version 0.0.2 of EllipticFEM. In which way do I need to change the sha file? Do I need to change the tags again?

**Comment by tkelman** [comment]
Is efc62eeeeb48cc0c76b13baf3f9a339e465f596a the commit you want to publish as 0.0.2? If so, edit the file at https://github.com/gerhardtulzer/METADATA.jl/edit/metadata-v2/EllipticFEM/versions/0.0.2/sha1.

I recommend using https://github.com/attobot/attobot next time around

**Comment by gerhardtulzer** [comment]
Yes, that's the version. I changed the sha file.

**Comment by tkelman** [comment]
When your tests run on Travis it looks like they're giving `ERROR: LoadError: LoadError: SystemError: opening file ./Gmsh saved files/gmshData.geo: No such file or directory` - do you get that locally from `Pkg.test("EllipticFEM")` ?

**Comment by gerhardtulzer** [comment]
I noticed a mistake in the code which caused that error (it was a problem with the folder structure). I corrected it and sent a new pull request.

**Pull request closed without merging**